graylog2-rest-models/src/main/java/org/graylog2/rest/models/system/loggers/responses/LogMessage.java

```diff
@@ -0,0 +1,39 @@
+package org.graylog2.rest.models.system.loggers.responses;
+
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+import java.util.List;
+
+@AutoValue
+@JsonAutoDetect
+public abstract class LogMessage {
+
+    @JsonProperty
+    public abstract String message();
+
+    @JsonProperty("class_name")
+    public abstract String className();
+
+    @JsonProperty
+    public abstract String level();
```

Comment:
Yes, exactly. Either use [`Level`](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Level.html) or write your own enum. This way you don't have to juggle with that stringly typed API. :wink: